components/carousel/template.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Template
+page_title: Template | Carousel for Blazor
+description: How to use the Template of the Carousel for Blazor.
+slug: carousel-template
+tags: telerik,blazor,carousel,template
+published: True
+position: 10
+---
+
+# Carousel Template
+
+The nested `<Template>` tag of the `<TelerikCarousel>` represents a standard [`RenderFragment`](https://docs.microsoft.com/en-us/aspnet/core/blazor/components/templated-components).
```

Comment:
Add a sentence to describe what you do with that template, and, more importantly, that it is mandatory.